includes/container/container.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Elementor;
+
+use Exception;
+use DI\ContainerBuilder;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor Container.
+ *
+ * Elementor container handler class is responsible for containerization
+ * manager classes and their dependencies.
+ *
+ * @since 1.0.0
+ */
+
+class Container extends ContainerBuilder {
+
+	protected static $instance;
+
+	public function __construct()
+	{
```

Comment:
AFAIK we write methods the following when the block opening brack is in the same line as the function definition:
E.g,
`	public function __construct() {`